### PR TITLE
chore: Add ui/modal stories

### LINKED ISSUE #893: Add ui/modal stories
Add Storybook stories for the component in the _modules/ui/modal_ folder:
- Modal

➡️  Create a `Default` story with args: `isOpen: true` and `children: [a div with lorem ipsum content]`. Disable `children` control with `argTypes: { children: { control: false } }`.

See http://localhost:6006/?path=/docs/ui-chip-chip--docs as an example story file.

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #893

## Demo
https://www.loom.com/share/b32f1233cb574e5b9b37c51a76cda17c

### CODE REVIEW

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you, a few minor comments

File: front/src/modules/ui/modal/components/__stories__/Modal.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const meta: Meta<typeof Modal> = {
+  title: 'UI/Modal/Modal',
+  component: Modal,
+  decorators: [ComponentDecorator],
+};
+export default meta;
+
+type Story = StoryObj<typeof Modal>;
+
+const Child = styled.div`
```
Comment (charlesBochet) [inline front/src/modules/ui/modal/components/__stories__/Modal.stories.tsx:17]: StyledContentContainer

File: front/src/modules/ui/modal/components/__stories__/Modal.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  color: ${({ theme }) => theme.font.color.primary};
+  margin: 5px;
```
Comment (charlesBochet) [inline front/src/modules/ui/modal/components/__stories__/Modal.stories.tsx:19]: let's add padding: theme.spacing(10)

**Comment by charlesBochet** [comment]
It looks like this PR is taking a lot of changes that have already been merged. Could you create a new branch and only cherry-pick the relavant commits?

**Comment by gitstart-twenty** [comment]
Hi @charlesBochet, sorry about that. Now the PR has only relevant changes, please take a look

**Pull request merged**